### ISSUE TITLE
no support for sorting resources by date

### ISSUE THREAD

**Issue by tomjscrypt** [microsoft/fhir-server#2252]
I tried to sort resources by date and I found out that only sorting by _lastUpdated is supported. I think it is quite important to be able to sort by date everywhere where date attribute occurs. 
Right now the only solution I see is to download all resources of given type, and then sort them on the client side.
When one can expect this feature being implemented?

**Comment by CaitlinV39** [comment]
@tomjscrypt: For Cosmos DB we also support firstName, lastName, and Clinicaldates on new databases. We dont plan to add any additional support on Cosmos DB. For SQL, we have PR #2169 which will add support for sorting on strings and dateTime fields. This has been in the works for a while, but we're planning to merge in the next couple of days.

**Comment by tomjscrypt** [comment]
> For SQL, we have PR #2169 which will add support for sorting on strings and dateTime fields. This has been in the works for a while, but we're planning to merge in the next couple of days.

Excellent news, will it work on Fhir Service Azure Healthcare Api as well?

**Comment by CaitlinV39** [comment]
Yes it will work on the FHIR service in the Azure Healthcare APIs :)
